app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class CommentsController < ApplicationController
+  before_action :find_post, only: %i(create destroy)
+
+  def index
+    @comment = Comment.all
+  end
+
+  def show
+    @comment = Comment.find params[:id]
+  end
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = @post.comments.create comment_params
+    if @comment.save
+      flash[:success] = t "comments.create"
+    else
+      flash.now[:warning] = t "comments.createfail"
+    end
+    redirect_to post_path @post
+  end
+
+  def destroy
+    @comment = @post.comments.find params[:id]
```

Comment:
I think you don't necessary this line
because you had defined find_post method

Reply:
![image](https://user-images.githubusercontent.com/45727724/93174248-3cf79100-f760-11ea-8c02-75109b5a2d66.png)
this error show when i delete the line